src/theme/tokens.ts

```diff
@@ -0,0 +1,156 @@
+// 1. Color Scale
+export const colorScale = {
+  // Gray
+  gray00: '#ffffff',
+  gray100: '#f7f8f9',
+  gray200: '#f3f4f5',
+  gray300: '#eeeff1',
+  gray400: '#dcdee3',
+  gray500: '#d1d3d8',
+  gray600: '#b0b3ba',
+  gray700: '#868b94',
+  gray800: '#555d6d',
+  gray900: '#2a3038',
+  gray1000: '#1a1c20',
+
+  // Yellow (Brand)
+  yellow00: '#fffef9',
+  yellow100: '#fffce5',
+  yellow200: '#fff8b7',
+  yellow300: '#fff38a',
+  yellow400: '#ffef5c',
+  yellow500: '#ffea2e',
+  yellow600: '#fee500',
+  yellow700: '#d5c000',
+  yellow800: '#ac9b00',
+  yellow900: '#847700',
+  yellow1000: '#5b5200',
+
+  // Brown
+  brown00: '#fff9f4',
+  brown100: '#ffeedc',
+  brown200: '#ffe2c4',
+  brown300: '#f9d0a8',
+  brown400: '#edbc8a',
+  brown500: '#cb9a69',
+  brown600: '#a97b4d',
+  brown700: '#875e35',
+  brown800: '#654321',
+  brown900: '#432a12',
+  brown1000: '#2d1b08',
+
+  // Blue
+  blue00: '#f8faff',
+  blue100: '#eff6ff',
+  blue200: '#e2edfc',
+  blue300: '#cbdffa',
+  blue400: '#aacefd',
+  blue500: '#85b8fd',
+  blue600: '#5e98fe',
+  blue700: '#217cf9',
+  blue800: '#135fcd',
+  blue900: '#0b4596',
+  blue1000: '#032451',
+
+  // Red
+  red00: '#fffafa',
+  red100: '#fdf0f0',
+  red200: '#fde7e7',
+  red300: '#fed4d2',
+  red400: '#feb7b3',
+  red500: '#fe928d',
+  red600: '#fc6a66',
+  red700: '#fa342c',
+  red800: '#ca1d13',
+  red900: '#921708',
+  red1000: '#4a1209',
+} as const;
+
+export type ColorScale = typeof colorScale;
```

Comment:
타입 선언 좋습니다~! 다만 제 개인적인 관점에서는 타입인 것을 직관적으로 알아볼 수 있도록 변수명 뒤에 `Type`을 붙이는 것이 좋을 것 같아요. 참고만 해주세요!
```suggestion
export type ColorScaleType = typeof colorScale;
```